src/core_plugins/input_control_vis/public/components/vis/range_control.js

```diff
@@ -32,38 +41,83 @@ export class RangeControl extends Component {
     this.setState({
       sliderValue: value,
       minValue: value.min,
-      maxValue: value.max
+      isMinValid: true,
+      maxValue: value.max,
+      isMaxValid: true,
+      errorMsgs: [],
     });
   }
 
   handleOnChangeComplete = (value) => {
     this.props.stageFilter(this.props.controlIndex, value);
   }
 
-  handleInputChange = (evt) => {
-    let inputValue = parseFloat(evt.target.value);
-    if (inputValue < this.props.control.min) {
-      inputValue = this.props.control.min;
-    } else if (inputValue > this.props.control.max) {
-      inputValue = this.props.control.max;
-    }
+  handleMinChange = (evt) => {
```

Comment:
Did the migration to EUI cause this to need to change?

Reply:
I found a bug in the master. Setting the min and max is totally broken in master. I fixed it in this PR because it was hard to judge how the min/max inputs worked/looked with the existing bug

Reply:
Gotcha, seems to work well now, so looks good to me!